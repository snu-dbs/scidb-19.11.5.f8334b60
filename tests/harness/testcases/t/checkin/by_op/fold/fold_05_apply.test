--setup
--start-query-logging

--test

_fold(apply(build(<v:int64>[i=3:5;j=7:9],i+j+1),w,10*v,x,100*v))
_fold(apply(build(<v:int64>[i=3:5:0:3;j=7:9],i+j+1),w,10*v,x,100*v))
_fold(apply(build(<v:int64>[i=3:5:0:2;j=7:9],i+j+1),w,10*v,x,100*v))
_fold(apply(build(<v:int64>[i=3:5:0:1;j=7:9],i+j+1),w,10*v,x,100*v))
_fold(apply(build(<v:int64>[i=3:5:0:1;j=7:9:0:3],i+j+1),w,10*v,x,100*v))

--cleanup
--stop-query-logging
