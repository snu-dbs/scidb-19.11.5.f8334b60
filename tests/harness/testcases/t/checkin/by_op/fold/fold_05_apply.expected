SCIDB QUERY : <_fold(apply(build(<v:int64>[i=3:5;j=7:9],i+j+1),w,10*v,x,100*v))>
{i} v,w,x,$v_1,$w_1,$x_1,$v_2,$w_2,$x_2
{3} 11,110,1100,12,120,1200,13,130,1300
{4} 12,120,1200,13,130,1300,14,140,1400
{5} 13,130,1300,14,140,1400,15,150,1500

SCIDB QUERY : <_fold(apply(build(<v:int64>[i=3:5:0:3;j=7:9],i+j+1),w,10*v,x,100*v))>
{i} v,w,x,$v_1,$w_1,$x_1,$v_2,$w_2,$x_2
{3} 11,110,1100,12,120,1200,13,130,1300
{4} 12,120,1200,13,130,1300,14,140,1400
{5} 13,130,1300,14,140,1400,15,150,1500

SCIDB QUERY : <_fold(apply(build(<v:int64>[i=3:5:0:2;j=7:9],i+j+1),w,10*v,x,100*v))>
{i} v,w,x,$v_1,$w_1,$x_1,$v_2,$w_2,$x_2
{3} 11,110,1100,12,120,1200,13,130,1300
{4} 12,120,1200,13,130,1300,14,140,1400
{5} 13,130,1300,14,140,1400,15,150,1500

SCIDB QUERY : <_fold(apply(build(<v:int64>[i=3:5:0:1;j=7:9],i+j+1),w,10*v,x,100*v))>
{i} v,w,x,$v_1,$w_1,$x_1,$v_2,$w_2,$x_2
{3} 11,110,1100,12,120,1200,13,130,1300
{4} 12,120,1200,13,130,1300,14,140,1400
{5} 13,130,1300,14,140,1400,15,150,1500

SCIDB QUERY : <_fold(apply(build(<v:int64>[i=3:5:0:1;j=7:9:0:3],i+j+1),w,10*v,x,100*v))>
{i} v,w,x,$v_1,$w_1,$x_1,$v_2,$w_2,$x_2
{3} 11,110,1100,12,120,1200,13,130,1300
{4} 12,120,1200,13,130,1300,14,140,1400
{5} 13,130,1300,14,140,1400,15,150,1500

